frontend/src/components/landingPage/MoreInformation.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <div id="moreInfoPage">
+    <app-header></app-header>
+    <h5></h5>
+    <div id="moreInfoDiv">
+      <div id="moreInfoTextDiv">
+        <p v-if="this.receivedId != this.errorStr">ID : {{ receivedId }}</p>
+        <p v-if="this.groupID != this.errorStr">Group ID : {{ groupID }}</p>
+        <p v-if="this.artifactID != this.errorStr">Artifact ID : {{ artifactID }}</p>
+        <p v-if="this.discovered != this.errorStr">Discovered : {{ discovered }}</p>
+        <p v-if="this.source != this.errorStr">Source : {{ source }}</p>
```

Comment:
Remove Source

Reply:
This has been fixed and commit has been pushed.